### DISCUSSION TITLE
integrate a variable

### DISCUSSION THREAD

**Discussion by MostafaMollaali** [pyvista/pyvista#2761] [Q&A] [answered]
Is there a way to integrate a variable in Pyvista that is equivalent to what "_[paraview.simple.IntegrateVariables](https://kitware.github.io/paraview-docs/latest/python/paraview.simple.IntegrateVariables.html)_" does in ParaView?

**Comment by MatthewFlamm** [comment, marked as answer]
I believe that this functionality isn't available yet.  There is [vtkIntegrateAttributes](https://vtk.org/doc/nightly/html/classvtkIntegrateAttributes.html), which could be used to implement this.

I'm going to transfer this to be an issue to track.

**Reply by MatthewFlamm** [reply]
See #2789
